tests/integration.rs

```diff
@@ -5557,12 +5596,12 @@ mod tests {
 
             assert!(pipesize >= PIPE_SIZE && pipesize1 >= PIPE_SIZE);
 
-            thread::sleep(std::time::Duration::new(60, 0));
-
             let mut child_dnsmasq = windows_guest.run_dnsmasq();
-            // Give some time for the guest to reach dnsmasq and get
-            // assigned the right IP address.
-            thread::sleep(std::time::Duration::new(30, 0));
+
+            // Wait to make sure Windows boots up
+            if !windows_guest.wait_for_boot() {
```

Comment:
Why don't you use `assert!()` instead in the `catch_unwind()` block?
```rust
let r = std::panic::catch_unwind(|| {
    assert!(windows_guest.wait_for_boot());
    ...
}
```

Reply:
Yeah, fixed as suggest, except one place where it needs to wait for boot for preliminary operations.

Thanks.